### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by DionnerFigueras** [leokhoa/laragon#633] [General]
Hello, I would like to start lagoon from a script (I know it's strange but although it may sound like an excuse, it's for a university project), well the thing is that I need the program to run and all the "Start All" services of automatically when you start the PC. I don't know if it's possible, but if it is I would greatly appreciate your help.

**Comment by Pen-y-Fan** [comment]
See the documentation [Docs screenshots \#preferences](https://laragon.org/docs/screenshots.html#Preferences)
